### PR TITLE
Options to guide codec selection

### PR DESCRIPTION
**Pull request by martinthomson**
This is pretty primitive, but I think that it is sufficient.

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -359,6 +359,16 @@
...
+
+          <dt>sequence&lt;RTCRtpCodecCapability&gt; codecs</dt>
+          <dd>
+            <p>This establishes what codecs the browser is permitted to
+            negotiate.  Codec capabilities that do not appear in this list
+            cannot be negotiated.</p>
+            <p>The order of this sequence determines the preference order.
+            Codecs that appear first in this list MUST be selected by the user
+            agent in preference to those that appear after.</p>
+          </dd>
```
Comment (alvestrand) [inline webrtc.html]: How does this work if this option is not specified?
I think the text should say that "if left off, browser can negotiate everything".

File: webrtc.html [hunk 2]
```diff
@@ -3312,8 +3322,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           </dd>
         </dl>
 
-        <dl class="idl" title="dictionary RTCRtpCodecCapability">
-          <dt>DOMString mimeType</dt>
+        <dl class="idl" title="interface RTCRtpCodecCapability">
+          <dt>readonly attribute DOMString mimeType</dt>
```
Comment (alvestrand) [inline webrtc.html]: Please explain the reasons for the dictionary -> interface change here.

Comment (martinthomson) [inline webrtc.html]: If this is not an interface, the application can construct preferred codecs that the browser didn't agree to supporting.

**Comment by martinthomson** [comment]
I'm going to drop this change, and await the merging of #293 before generating a new PR.

**Pull request closed without merging**